source/includes/_enrichment.md

```diff
@@ -35,23 +37,25 @@ url = "https://api.apollo.io/api/v1/people/bulk_match"
 
 data = {
     "api_key": "YOUR API KEY HERE",
-    "reveal_personal_emails": true,
+    "reveal_personal_emails": True,
```

Comment:
Please keep this as true instead of True. Same way for other places as well

Reply:
This is actually wrong. Python boolean representation is `True `and not `true`
Therefore, when a request is made from python, we should have True.